pyext/src/restraints/crosslinking.py

```diff
@@ -455,6 +455,14 @@ def get_output(self):
 
         return output
 
+    def get_likelihood(self):
+        """Get the likelihood value for the restraint"""
```

Comment:
This is just restating the method name. Maybe something like `"""Get the probability of the (unweighted) restraint being satisfied"""` ?

Reply:
Updated to `"""Get the unweighted likelihood of the restraint.""" `